README.md

```diff
@@ -16,7 +16,14 @@ One of the dangers with an auto-provisioning course approach is the proliferatio
     use.test.url=true  
     esb.url=esb server e.g. https://api-qa.its.umich.edu:9443/store/  
     api.call.type=1 or 2  1 means canvas call 2 means canvas call via ESB
+    term1=1990;08/12/2014
+    term2=2000;08/12/2014
+    .....
+    termN=2010;01/04/2015
+    term.count=N
     ```
+    
+    
    
 ## Run Directions
```

Comment:
In productions it shouldn't run out of the target directory.  It should be installed separately.